src/vsr/replica.zig

```diff
@@ -5269,7 +5269,10 @@ pub fn ReplicaType(
                 } else if (entry.session < message.header.session) {
                     // This cannot be because of a partition since we check the client's view
                     // number.
-                    log.err("{}: on_request: ignoring newer session (client bug)", .{self.replica});
+                    log.warn(
```

Comment:
This feel like it warrants an `.err` --- if this _is_ a bug, we'd want our users to report this case (but this probably shouldn't be an outright assert, as we dno't want to trust clients __too__ much. Server is running with ECC RAM, but clients dont)

Reply:
this applies to other "bug" cases